spot_rl_experiments/spot_rl/envs/base_env.py

```diff
@@ -42,6 +42,7 @@
 from std_msgs.msg import Float32, String
 
 MAX_CMD_DURATION = 5
+TRAVLE_TIME_JOINT_BASE_ARM_CONTROL = 5
```

Comment:
@jimmytyyang @tusharsangam we already have a parameter in config.yaml :: `ARM_TRAJECTORY_TIME_IN_SECONDS`. Can we re-use that instead?